native/packages/react-native-bpk-component-horizontal-nav/readme.md

```diff
@@ -70,6 +70,7 @@ export default class App extends Component {
 | onPress             | func                                  | true     | -             |
 | title               | string                                | true     | -             |
 | accessibilityLabel  | string                                | false    | props.title   |
+| small               | bool                                  | false    | false         |
```

Comment:
Maybe we should be more explicit about what's actually `small`? It's a bit ambigious as is, something like `smallText` would be a bit clearer IMO